<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>video_playback</class>
 <widget class="QMainWindow" name="video_playback">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>start_car</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background-image: url('/home/phong/Desktop/antitheft/GUI/3/a.jpg');
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,2">
    <item>
     <widget class="QPushButton" name="back_button">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:white;
border: 3px solid red;</string>
      </property>
      <property name="text">
       <string>Back to Review Footage</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QVideoWidget" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,5">
      <item>
       <widget class="QPushButton" name="play_button">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>22</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
border: 3px solid red;</string>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="time_status">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
background-color: white;
    border: 1px solid;
    height: 40px;
    margin: 0px;
    }
QSlider::handle:horizontal {
	background-color: red;
     border: 1px solid;
    height: 40px;
    width: 40px;
    margin: -15px 0px;
    }</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
